ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -328,6 +328,35 @@ class FamilyReads extends React.PureComponent {
     )
   }
 
+  getSampleColors = () => {
+    const { openFamily } = this.state
+    const { igvSamplesByFamilySampleIndividual, individualsByGuid } = this.props
+    const igvSampleIndividuals = (
+      openFamily && (igvSamplesByFamilySampleIndividual || {})[openFamily]) || {}
+    return Object.keys(igvSampleIndividuals[GCNV_TYPE] || {}).reduce(
+      (acc, iGuid) => ({
+        ...acc,
+        [iGuid]: individualsByGuid[iGuid].affected === AFFECTED ? 'red' : 'blue',
```

Comment:
this needs to share code with the function we use when we compute the track types, so if we ever want to change the colors we only need to change one place. Maybe a helper function like `getSampleColor(individual)`